lab/03-testing.md

```diff
@@ -66,12 +65,12 @@ Navigate to the `demo-code` folder and run the `test` command:
 
 ## Testing via GitHub Actions
 
-This is great, however we don't want to need to remember to run these tests every time we make a change. We can use GitHub Actions to run these tests for us.
+This is great, however we may forget to run these tests every time we make a change. We can use GitHub Actions to run these tests for us.
```

Comment:
The previous sentence was correct grammar, in retrospect, but I was having a hard time parsing it. Other suggestions welcome.